packages/vitest/src/runtime/collect.ts

```diff
@@ -34,7 +33,7 @@ export async function collectTests(paths: string[], config: ResolvedConfig) {
       return await import(`${filepath}?v=${hash}`)
   }
 
-  for (const filepath of paths) {
+  for (const filepath of matchPattern(paths, config.fileNamePattern)) {
```

Comment:
We don't need this. Files are already filtered by Vitest by this point.

Reply:
OK, without that I cannot filter the initial test run, to check functionality in `pnpm test:ci`
I will think about how to deal with it later. 

Thank you